qiskit/providers/models/backendconfiguration.py

```diff
@@ -47,11 +75,11 @@ class BackendConfigurationSchema(BaseSchema):
     local = Boolean(required=True)
     simulator = Boolean(required=True)
     conditional = Boolean(required=True)
-    open_pulse = Boolean(required=True)
     memory = Boolean(required=True)
     max_shots = Integer(required=True, validate=Range(min=1))
 
     # Optional properties.
+    open_pulse = Boolean()
```

Comment:
If both subclasses define `open_pulse` as required (and actually it is `required` and provided by the backends), can you leave it as `required`?